apps/web/ui/domains/register-domain-form.tsx

```
@@ -347,6 +332,20 @@ export function RegisterDomainForm({
           />
         )}
       </div>
+      {searchedDomain && showTerms && (
+        <p className="mt-4 text-pretty text-center text-xs text-neutral-500">
+          By claiming your .link domain, you agree to our{" "}
+          <a
+            href="https://dub.co/help/article/free-dot-link-domain#terms-and-conditions"
+            target="_blank"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)